content/post/know-your-aws-community-day-india-2020-speaker-swaroop-hegde.md

```diff
@@ -0,0 +1,22 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T15:48:24Z
+description = "Swaroop Hegde will be speaking on \"Using Aurora Global Database for 10x Faster API\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/swaroop-hegde.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker", "Aurora"]
+title = "Know Your AWS Community Day India 2020 Speaker - Swaroop Hegde"
+
++++
+Swaroop Hegde is a Co-founder at BlockVigil, USA will be speaking on "**Using Aurora Global Database for 10x Faster API**" from 10:00 AM - 10:45 AM Track 1 on 16th October 2020 at AWS Community Day India 2020.
```

Comment:
>  working as [...] will be speaking on 


is not grammatically right. This should be 

`<name>` is currently `<title>` at `<company>`. `<First name>` will be speaking on <session title>....

Also, no need to mention "USA",